### DISCUSSION TITLE
history navigation on "browse code"

### DISCUSSION THREAD

**Discussion by jimmywarting** [npm/feedback#946] [Website] [answered]
the new code tab to view the source is grate, but one mistake i always keep doing is using the navigation history to go back.
but given that npm is a SPA and the url never change when clicking around all the files makes it so that i leave the page.

would be grate to have a added `file=path` key/value added to the url
```
https://www.npmjs.com/package/<package>?activeTab=code&file=xyz
                                                       ^
```
it would then also be better shareable.

**Comment by monishcm** [comment, marked as answer]
Thank you for your feedback. This is a feature we would love to ship but will take time as we are focusing on other priorities.
